builtin/client/register.lua

```diff
@@ -63,6 +63,7 @@ core.registered_on_mods_loaded, core.register_on_mods_loaded = make_registration
 core.registered_on_shutdown, core.register_on_shutdown = make_registration()
 core.registered_on_receiving_chat_message, core.register_on_receiving_chat_message = make_registration()
 core.registered_on_sending_chat_message, core.register_on_sending_chat_message = make_registration()
+core.registered_on_chatcommand, core.register_on_chatcommand = make_registration()
```

Comment:
Shouldn't this be core.registered_on_chatcommand**s**?

Reply:
Maybe, I'm honestly not sure though. Originally I made it singular only because `core.registered_on_sending_chat_message` was singular and I thought it might be more important to be consistent with that. But if I'm missing something or if `registered_on_sending_chat_message` should really be plural itself, then I'll happily change this to `registered_on_chatcommands`.